IAM/component.yaml

```diff
@@ -9,46 +9,46 @@ satisfies:
   covered_by: []
   implementation_status: none
   narrative: 'Account log out is set to 15 minutes of inactivity within the IAM console
-    per accountwithin the 18F virtual infrastructure.
+    per account within the organization virtual infrastructure.
 
     '
   standard_key: NIST-800-53
 - control_key: AC-2 (1)
   covered_by: []
   implementation_status: none
   narrative: 'AWS infrastructure as a service Management Life Cycle is automated to
-    use AWS CLI scripts. 18F AWS Virtual Private Cloud can use the AWS Command Line
+    use AWS CLI scripts. The organization AWS VPC can use the AWS Command Line
     Interface (CLI) to automate the account management LifeCycle within its envoriment.
-    18F uses the AWS IAM console for semi-automated automated account manamgemt
+    The organization uses the AWS IAM console for semi-automated automated account manamgemt
 
     '
   standard_key: NIST-800-53
 - control_key: AC-6
   covered_by: []
   implementation_status: none
   narrative: 'IAM policies are attached to the users, enabling centralized control
-    of permissions for users under 18F AWS Account to access services, buckets or
-    objects. With IAM policies, 18F only grant users within its own AWS account permission
+    of permissions for users under the ogranizations AWS Account to access services, buckets or
+    objects. With IAM policies, the ogranizations only grant users within its own AWS account permission
     to access its Amazon resources.
 
-    18F AWS IAM policies are defined to grant only the required access for 18F staff
-    necessary to perform their functions. 18F defines least privilege access to each
+    AWS IAM policies are defined to grant only the required access for the ogranization staff
+    necessary to perform their functions. The organizatioT defines least privilege access to each
     user, group or role.
 
     Security functions within the AWS infrastructure are explicitly defined within
     IAM to include read-only permissions for any user functions.
 
-    18F incorporate running the IAM Policy Simulator to test policies for least privilege
+    The organization incorporate running the IAM Policy Simulator to test policies for least privilege
     access for users and groups.
 
     '
   standard_key: NIST-800-53
 - control_key: AC-6 (1)
   covered_by: []
   implementation_status: none
-  narrative: '18F explicitly authorizes access to administrative and security functions
-    of its virtual infrastructure and Cloud.Gov platform to designated individuals
-    within the Devops and SecOps team.  No other authrozations to security and administrative
+  narrative: 'The organization explicitly authorizes access to administrative and security functions
+    of its virtual infrastructure and residing platforms to designated individuals
+    within the organizations SecOps and DevOps team.  No other authorizations to security and administrative
```

Comment:
Hmm, probably a larger conversation, but I'm not sure that we are going to be able to refer to "the organization's SecOps and DevOps team", since this documentation supposed to be generally applicable. Not sure how best to handle that.